tslint.json

```diff
@@ -73,5 +73,6 @@
     "valid-typeof": true,
     "variable-name": [true, "check-format", "allow-leading-underscore", "ban-keywords"],
     "ordered-imports": [true]
+    // "object-literal-key-quotes": [true, "as-needed"]
```

Comment:
We never had that option. Note how I added it. I know I shouldn't have put this in this PR.